docs/src/main/asciidoc/_templates/template-reference.adoc

```diff
@@ -4,14 +4,15 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 ////
-TODO: See xref:{doc-guides}/doc-reference.adoc[Quarkus style and content guidelines] for details
-- Title should have an implied "About..." in front. 
-- ID should end with '-reference' (using the filename works)
-- choose appropriate categories
+TODO: 
+- Title: https://quarkus.io/guides/doc-reference#titles-headings
+- Use the file name as the ID
+- Choose appropriate categories: https://quarkus.io/guides/doc-reference#categories
```

Comment:
```suggestion
- Choose appropriate categories: link:https://quarkus.io/guides/doc-reference#categories
```

Reply:
Do I need this? It is in a comment in the template header... not rendered or kept.